### DISCUSSION TITLE
BITMAPV5HEADER is missing?

### DISCUSSION THREAD

**Discussion by EchoEllet** [halildurmus/win32#890] [General]
To clarify, I'm new to Win32 API (itself and not the Dart package) and Dart FFI.

I was looking in the Win32 Docs (Of Windows) and I found that [BITMAPV5HEADER](https://learn.microsoft.com/en-us/windows/win32/api/wingdi/ns-wingdi-bitmapv5header) is not available as part of `win32` package while [BITMAPINFOHEADER](https://learn.microsoft.com/en-us/windows/win32/api/wingdi/ns-wingdi-bitmapinfoheader) is available.

Assuming that `BITMAPINFOHEADER` is generated, maybe it didn't generate the `BITMAPV5HEADER` for a reason or it's controlled manually somewhere that define how those will be generated?

**Comment by halildurmus** [comment]
> Assuming that `BITMAPINFOHEADER` is generated, maybe it didn't generate the `BITMAPV5HEADER` for a reason or it's controlled manually somewhere that define how those will be generated?

That's correct. Currently, [JSON files](https://github.com/halildurmus/win32/tree/main/packages/generator/data) determine which APIs are projected by the [generator](https://github.com/halildurmus/win32/tree/main/packages/generator).

I plan to eliminate these JSON files and fully project the entire Win32 API in the future (https://github.com/halildurmus/win32/issues/819).

**Reply by EchoEllet** [reply]
Thank you for the clarification.
